### PR TITLE
Issue #4868 - Adding warning message to db_connect when already connected

### PR DESCRIPTION
**Pull request by nstarke**
Issue #4868 - Adding warning message to db_connect when already connected.

### CODE REVIEW

File: lib/msf/ui/console/command_dispatcher/db.rb [hunk 1]
```diff
@@ -1706,6 +1706,15 @@ def cmd_db_connect_help
...
   def cmd_db_connect(*args)
     return if not db_check_driver
+    if framework.db.connection_established?
+      cdb = ""
+      ::ActiveRecord::Base.connection_pool.with_connection { |conn|
+        if conn.respond_to? :current_database
+          cdb = conn.current_database
+        end
+      } 
+      return print_status("#{framework.db.driver} already connected to #{cdb}")
```
Comment (shuckins-r7) [inline lib/msf/ui/console/command_dispatcher/db.rb]: Sweet, thanks @nstarke! One minor comment here: might want to follow this with something along the lines of "Run db_disconnect first if you wish to connect to a different DB."

**Comment by wvu-r7** [comment]
@nstarke: https://github.com/blog/1506-closing-issues-via-pull-requests. :)

**Comment by wvu-r7** [comment]
Also, putting the issue# at the start of the title is a bit confusing.

**Comment by nstarke** [comment]
Thanks for the tips on pull requests; I will be sure to follow these going forward.

**Pull request merged**